node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+
+function readAll() {
+  const [, , ...filePaths] = process.argv;
+  filePaths.forEach((path) => {
+    if (!path.includes('.txt')) {
```

Comment:
You might use `.endsWith` instead to make sure you are only checking the file extension.